corehq/ex-submodules/couchexport/writers.py

```diff
@@ -568,11 +568,29 @@ def parse_feature(coordinates, properties):
             "properties": properties,
         }
 
+    @staticmethod
+    def _find_geo_property_by_path(table):
+        # For form exports we store the path to the geo property, so this is an
+        # attempt to find the header by the path specification
+        is_path = len(table.selected_geo_property.split('.')) > 1
```

Comment:
This is a heuristic to check if the string is something like a path (e.g. "form.property"), otherwise it's not a path.

Reply:
nit: Could it simply be `'.' in table.selected_geo_property`?

Reply:
Though we need this though? You have handled this in `get_column_by_path_str` as well?

Reply:
You're right. We don't need it.